common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -2892,3 +2894,26 @@ func (d *cassandraPersistence) RangeDeleteReplicationTaskFromDLQ(
 	}
 	return nil
 }
+
+func (d *cassandraPersistence) CreateFailoverMarkerTasks(
+	request *p.CreateFailoverMarkersRequest,
+) error {
+
+	batch := d.session.NewBatch(gocql.LoggedBatch)
+	for _, task := range request.Markers {
+		t := []p.Task{task}
+		if err := createReplicationTasks(
+			batch,
+			t,
+			d.shardID,
+			task.DomainID,
+			rowTypeReplicationWorkflowID,
+			rowTypeReplicationRunID,
+			task.GetVisibilityTimestamp().UnixNano(),
+		); err != nil {
+			return err
+		}
+	}
+
+	return d.session.ExecuteBatch(batch)
```

Comment:
Please convert common gocql error to `ServiceBusyError`, `InternalServiceError` etc.

Reply:
Handled the gocql errors